lib/node_modules/@stdlib/ndarray/iter/README.md

```diff
@@ -71,12 +71,151 @@ var o = ns;
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+// Example: Get an array consisting of keys/properties inside ns.
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( '@stdlib/ndarray/iter' );
 
 console.log( objectKeys( ns ) );
 ```
 
+```javascript
+// Example: Iterate over [index, column] pairs for each column in a matrix
+var ndarray = require( '@stdlib/ndarray/ctor' );
+var ns = require( '@stdlib/ndarray/iter' );
+
+var array2D = [ [ 1, 2, 3 ], [ 4, 5, 6 ], [ 7, 8, 9 ] ];
+var data = [].concat.apply( [], array2D );
+var x = ndarray( 'generic', data, [ 3, 3 ], [ 3, 1 ], 0, 'row-major' );
+var iterEntries = ns.nditerEntries( x );
+var entry;
+
+for ( entry of iterEntries ) {
+    console.log( 'Index:', entry[ 0 ], 'Value:', entry[ 1 ] );
+}
+```
+
+```javascript
+// Example: Iterate over each row of the ndarray
```

Comment:
These examples seem to just be pulling from the READMEs of the child packages. I think we would prefer something fresh, which ideally combines the functionality of multiple packages to demonstrate how they can all work together.